requests/adapters.py

```diff
@@ -418,11 +421,14 @@ def send(self, request, stream=False, timeout=None, verify=True, cert=None, prox
                        "timeout tuple, or a single float to set "
                        "both timeouts to the same value".format(timeout))
                 raise ValueError(err)
-        else:
+        elif timeout is not None:
             timeout = TimeoutSauce(connect=timeout, read=timeout)
```

Comment:
Hmm. Is it really breaking?
Looking at urllib3, leaving out the timeout argument will result in calling ``connectionpool.HTTPConnectionPool._get_timeout(_Default)``, which will use ``Timeout.DEFAULT_TIMEOUT`` if the ``HTTPConnectionPool`` had been initialized without a timeout. And ``Timeout.from_float(DEFAULT_TIMEOUT)`` yields the same result as if it had been called with ``None``: https://github.com/shazow/urllib3/blob/master/urllib3/util/timeout.py#L117

So the results should be the same...